client/config/config.go

```diff
@@ -265,6 +265,11 @@ type Config struct {
 	//
 	// This configuration is only considered if no host networks are defined.
 	BindWildcardDefaultHostNetwork bool
+
+	// ExternalGetterCommand configures an external tool uesd to retrieve artifacts.
+	// If not such command is defined (as is default), Nomad falls back and uses its
+	// internal version of go-getter.
+	ExternalGetterCommand string
```

Comment:
I personally think this should be a per-job setting rather than a client-wide setting - possibly allow-listing the commands allowed to be used in the client instead

Reply:
yeah it would make sense to be able to override this on a per-job basis - can add that in a second PR. I would keep it as a client-wide setting as well, as there are cases were this is implements a platform feature that users should not need to care about.